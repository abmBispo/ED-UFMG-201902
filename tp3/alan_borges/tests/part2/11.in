128
letras inuteis e letras dispensaveis dizia ele que servico diverso prestam o d e o t tem quase o mesmo som o mesmo digo do b e do p o mesmo do s do c e do z o mesmo do k e do g etc sao trapalhices caligraficas veja os algarismos nao ha dous que facam o mesmo oficio e e e e admire a beleza com que um e um formam esta cousa que se exprime por agora dobre e tera multiplique por igual numero da e assim por diante mas onde a perfeicao e maior e no emprego do zero o valor do zero e em si mesmo nada mas o oficio deste sinal negativo e justamente aumentar um sozinho e um ponha lhe dous
c e
c do
c mesmo
c o
c por
c que
c um
c a
c dous
c letras
c mas
c oficio
c zero
c admire
c agora
c algarismos
c assim
c aumentar
c b
c beleza
c c
c caligraficas
c com
c cousa
c d
c da
c deste
c diante
c digo
c dispensaveis
c diverso
c dizia
c dobre
c ele
c em
c emprego
c esta
c etc
c exprime
c facam
c formam
c g
c ha
c igual
c inuteis
c justamente
c k
c lhe
c maior
c multiplique
c nada
c nao
c negativo
c no
c numero
c onde
c os
c p
c perfeicao
c ponha
c prestam
c quase
c s
c sao
c se
c servico
c si
c sinal
c som
c sozinho
c t
c tem
c tera
c trapalhices
c valor
c veja
c z
