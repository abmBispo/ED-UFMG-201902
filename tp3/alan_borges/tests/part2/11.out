110
1110
0000
1111
00010
00011
00100
001010
001011
001100
001101
001110
001111
0100000
0100001
0100010
0100011
0100100
0100101
0100110
0100111
0101000
0101001
0101010
0101011
0101100
0101101
0101110
0101111
0110000
0110001
0110010
0110011
0110100
0110101
0110110
0110111
0111000
0111001
0111010
0111011
0111100
0111101
0111110
0111111
1000000
1000001
1000010
1000011
1000100
1000101
1000110
1000111
1001000
1001001
1001010
1001011
1001100
1001101
1001110
1001111
1010000
1010001
1010010
1010011
1010100
1010101
1010110
1010111
1011000
1011001
1011010
1011011
1011100
1011101
1011110
1011111
